_services_projects/forest-service.md

```diff
@@ -1,9 +1,9 @@
 ---
-agency: Forest Service
+agency: U.S. Forest Service
 title: Moving land permits online
 subtitle: Easing access to public lands
 permalink: /what-we-deliver/forest-service/
-excerpt: Using prototypes and modular contracting to help the Forest Service buy an online permit system.
+excerpt: Using prototypes and modular contracting to help the Forest Service develop an online permitting system.
```

Comment:
I think we want to say both `buy` and `devop` because this is a system that was developed mostly from modular contracts and i think that is a big part of this engagement's story.